src/components/dialogmote/referat/Buttonrow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+import { Flatknapp, Hovedknapp, Knapp } from "nav-frontend-knapper";
+
+const texts = {
+  send: "Lagre og send",
+  preview: "Se forhåndsvisning",
+  abort: "Avbryt",
+};
+
+interface KnapperadProps {
+  sendMethod: () => void;
+  previewMethod: () => void;
+  abortMethod: () => void;
+}
+
+const KnapperadWrapper = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  padding-top: 1em;
+  flex-wrap: wrap;
+
+  > * {
+    margin-right: 0.5em;
+    margin-left: 0;
+  }
+`;
+
+const Buttonrow = ({
+  sendMethod,
+  previewMethod,
+  abortMethod,
+}: KnapperadProps) => {
+  return (
+    <KnapperadWrapper>
```

Comment:
Mulig du bør sette riktig htmlType på hver knapp ("submit" eller "button") for å sikre at bare hovedknapp submitter formet de ligger inni.

Reply:
Det er lurt! Alle submitter nå, og det er jo ikke helt heldig 😅